docs/src/pages/components/RatingBar.vue

```diff
@@ -0,0 +1,299 @@
+<template>
+  <page-content page-title="Components - Rating Bar">
+    <docs-component>
+      <div slot="description">
+        <p>Rating Bar can be used to show a rating to user or capture a rating from user.</p>
+        <p>The following classes can be applied to change the color palette:</p>
+        <ul class="md-body-2">
+          <li><code>md-primary</code></li>
+          <li><code>md-accent</code></li>
+          <li><code>md-warn</code></li>
+        </ul>
+      </div>
+
+      <div slot="api">
+        <api-table name="md-rating-bar">
+          <md-table slot="properties">
+            <md-table-header>
+              <md-table-row>
+                <md-table-head>Name</md-table-head>
+                <md-table-head>Type</md-table-head>
+                <md-table-head>Description</md-table-head>
+              </md-table-row>
+            </md-table-header>
+
+            <md-table-body>
+              <md-table-row>
+                <md-table-cell>v-model</md-table-cell>
+                <md-table-cell><code>Number</code></md-table-cell>
+                <md-table-cell>A required model object to bind the value. The value is limited by range [0, 1].</md-table-cell>
+              </md-table-row>
+
+              <md-table-row>
+                <md-table-cell>md-num-stars</md-table-cell>
```

Comment:
I feel like the property name `md-num-stars` is a little confusing. I would suggest something like `md-max-rating` or `md-max`. Then make note that this should be a _whole_ number.